watertap/property_models/anaerobic_digestion/modified_adm1_reactions.py

```diff
@@ -2178,17 +2163,13 @@ def rule_I_nh3(self):
         )
 
         def rule_I_pH_aa(self):
-            return pyo.Expr_if(
-                self.pH > self.params.pH_UL_aa,
-                1,
-                pyo.exp(
-                    -3
-                    * (
-                        (self.pH - self.params.pH_UL_aa)
-                        / (self.params.pH_UL_aa - self.params.pH_LL_aa)
-                    )
-                    ** 2
-                ),
+            return (
+                -3
```

Comment:
Same comments as before- should rename expressions to indicate logarithmic form as well as doc string descriptions.